README.md

```diff
@@ -441,6 +441,7 @@ And to install `liquidctl` into the environment:
 ```
 python -m pip install .
 ```
+_OPTIONAL: When installing from source, you might want to try prepending sudo to this last command, in case your .service file returns driver error_
```

Comment:
While I understand what you're trying to achieve here, this isn't the place for it: the note you proposed is Linux-specific and, even then, assumes systemd and that a `.service` file is in use.

Additionally, I think the behavior or `sudo pip install` within a virtual environment, which we also recommend a few paragraphs above, is not going to be what the user wants/expects.

Also, this goes beyond privileged/unprivileged users: liquidctl needs to be installed somewhere in the `PATH` used by the service, which can vary depending on which user the service has been set up to run as. IIRC there's already a note about `PATH` in the automation section, but maybe it isn't obvious enough (or it was moved elsewhere at some point).

Basically, this needs to added to the documentation, yes, but with a bit more care. For now, please remove this change from this PR and open a new one just for that discussion. Let's not pollute the subject of supporting a new device with non-driver-specific documentation changes.